Rakefile

```diff
@@ -177,3 +146,30 @@ rescue LoadError
     raise 'Install github_changelog_generator to get access to automatic changelog generation'
   end
 end
+
+desc 'Run acceptance tests'
+task :acceptance do
+
+  begin
+    if ENV['MATRIX_TARGET']
```

Comment:
out of interest, will `MATRIX_TARGET` be one of `puppet6/7` from ci.yml?

Reply:
Correct